docs/source/concepts/bento.rst

```diff
@@ -399,7 +399,7 @@ Description
 
 ``description`` field allows user to customize documentation for any given Bento.
 
-The description contents must be plain text, optionally in `Markdown <https://daringfireball.net/projects/markdown/syntax>`_ format. Description
+The description contents support Jinja2 Template, optionally in `Markdown <https://daringfireball.net/projects/markdown/syntax>`_ format. Description
```

Comment:
I believe that this is way too complex for it to be in concept page.

Maybe you could create an advanced section for this in `advanced guides`?